lib/cli.ex

```diff
@@ -38,6 +38,15 @@ defmodule Onigumo.CLI do
   end
 
   defp usage_message() do
+    IO.puts("""
+    onigumo: invalid usage
+    Usage: onigumo [OPTION]... [COMPONENT]
+
+    Try `onigumo --help' for more options.
+    """)
```

Comment:
This produces an excessive newline. `IO.puts` puts (😃) a `\n` at the end automatically. It happens for the usage message too though. Not in scope of this PR. Created https://github.com/Glutexo/onigumo/issues/266.